articles/migrations/guides/extensibility-node8.md

```diff
@@ -1,57 +1,67 @@
 ---
-title: Migration Guide - Extensibility and Node.js v8
+title: Migration Guide - Extensibility and Node 8
 description: This article covers the Auth0 features/modules affected, as well as our recommendations to ensure a smooth migration process.
 toc: true
 ---
-# Migration Guide: Extensibility and Node.js v8
+# Migration Guide: Extensibility and Node 8
 
 :::warning
-The Node.js 8 runtime will be available to all public cloud customers on **April 17th, 2018**.
+The Node 8 runtime will become available to all public cloud customers on **April 17, 2018**.
 :::
 
-Beginning 2018 April 30, [Node.js v4 will be going out of long-term support (LTS)](https://github.com/nodejs/Release#release-schedule), which means that the Node.js development team will no longer be back-porting critical security fixes to this version.
+Beginning April 30, 2018, [Node.js v4 will be going out of long-term support (LTS)](https://github.com/nodejs/Release#release-schedule), which means that the Node.js development team will no longer be back-porting critical security fixes to this version and this _could_ expose your extensibility code to security vulnerabilities.
 
-As such, Auth0 will be migrating from Node.js v4 to Node.js v8.
+As such, Auth0 will be migrating from Node 4 to Node 8.
+
+:::note
+We will **NOT** be shutting down the Node 4 runtime after the April 30 LTS deadline. Your extensibility code will continue to run on Node 4, if you choose not to upgrade to Node 8 at this time. After April 30, you will assume the risk of potential security issues if you choose to continue with Node 4.
+:::
 
 In this document, we:
 
 * Provide recommendations on how you can ensure a smooth migration for your environment
 * Detail the specific modules effected
 
+
+
+
```

Comment:
This blank space doesn't really affect anything, the result looks the same with this (that only has 1 empty link) https://github.com/auth0/docs/pull/6017/files#diff-c3833e5f97719d03a1895ec032b31baeR44 if you check out the review app